kratos/python_scripts/assign_vector_variable_to_entities_process.py

```diff
@@ -65,41 +63,18 @@ def __init__(self, Model, settings ):
 
         import assign_scalar_variable_to_entities_process
 
-        # Component X
-        if not settings["value"][0].IsNull():
-            x_params = KratosMultiphysics.Parameters("{}")
-            x_params.AddValue("model_part_name",settings["model_part_name"])
-            x_params.AddValue("mesh_id",settings["mesh_id"])
-            x_params.AddValue("interval",settings["interval"])
-            x_params.AddValue("value",settings["value"][0])
-            x_params.AddEmptyValue("variable_name").SetString(settings["variable_name"].GetString() + "_X")
-            x_params.AddValue("local_axes",settings["local_axes"])
-            x_params.AddValue("entities",settings["entities"])
-            self.aux_processes.append( assign_scalar_variable_to_entities_process.AssignScalarVariableToEntitiesProcess(Model, x_params) )
-
-        # Component Y
-        if not settings["value"][1].IsNull():
-            y_params = KratosMultiphysics.Parameters("{}")
-            y_params.AddValue("model_part_name",settings["model_part_name"])
-            y_params.AddValue("mesh_id",settings["mesh_id"])
-            y_params.AddValue("interval",settings["interval"])
-            y_params.AddValue("value",settings["value"][1])
-            y_params.AddEmptyValue("variable_name").SetString(settings["variable_name"].GetString() + "_Y")
-            y_params.AddValue("local_axes",settings["local_axes"])
-            y_params.AddValue("entities",settings["entities"])
-            self.aux_processes.append( assign_scalar_variable_to_entities_process.AssignScalarVariableToEntitiesProcess(Model, y_params) )
-
-        # Component Z
-        if not settings["value"][2].IsNull():
-            z_params = KratosMultiphysics.Parameters("{}")
-            z_params.AddValue("model_part_name",settings["model_part_name"])
-            z_params.AddValue("mesh_id",settings["mesh_id"])
-            z_params.AddValue("interval",settings["interval"])
-            z_params.AddValue("value",settings["value"][2])
-            z_params.AddEmptyValue("variable_name").SetString(settings["variable_name"].GetString() + "_Z")
-            z_params.AddValue("local_axes",settings["local_axes"])
-            z_params.AddValue("entities",settings["entities"])
-            self.aux_processes.append( assign_scalar_variable_to_entities_process.AssignScalarVariableToEntitiesProcess(Model, z_params) )
+        list_directions = ["_X", "_Y", "_Z"]
+        for i_dir in range(3):
```

Comment:
```suggestion
        for i_dir, dir in enumerate(["_X", "_Y", "_Z"]):
```

this is basically the only review comment.
can you please check if this is also applicable for the other processes?